### PR TITLE
Vendor gardener/gardener@v1.11.1

### PR DESCRIPTION
**Pull request by ialidzhikov**
**How to categorize this PR?**

/area open-source
/kind task
/priority normal
/platform alicloud

**What this PR does / why we need it**:
Vendor gardener/gardener@v1.11.1

**Release note**:

```improvement operator
`github.com/gardener/gardener` dependency is now updated to `v1.11.1`.
```

### CODE REVIEW

**Review by vpnachev — APPROVED** [review]
/lgtm

**Review by minchaow — APPROVED** [review]
/lgtm

File: Makefile [hunk 1]
```diff
@@ -130,11 +130,11 @@ format:
 
 .PHONY: test
 test:
-	@$(REPO_ROOT)/vendor/github.com/gardener/gardener/hack/test.sh ./cmd/... ./pkg/...
+	@SKIP_FETCH_TOOLS=1 $(REPO_ROOT)/vendor/github.com/gardener/gardener/hack/test.sh ./cmd/... ./pkg/...
```
Comment (minchaow) [inline Makefile:133]: what's the purpose of this change?

Comment (timebertt) [reply Makefile:133]: It skips fetching the binaries needed for envtest integration tests (etcd, kube-apiserver, kubectl) (see https://github.com/kubernetes-sigs/controller-runtime/tree/master/pkg/envtest and https://github.com/gardener/gardener/blob/master/hack/setup-envtest.sh).
Currently, there are no envtest test here, I assume. So no need, to fetch those binaries.

Comment (minchaow) [reply Makefile:133]: thanks for explanation.
